src/providers/address/address.ts

```diff
@@ -19,6 +19,11 @@ export class AddressProvider {
     this.core = this.bwcProvider.getCore();
   }
 
+  public translateToCashAddress(addressToTranslate: string): string {
```

Comment:
Could you add a log like this: Converted: "legacyAddress" -> "newCashAddress"

Reply:
yes!